text/0000-clarity-api-description.md

```diff
@@ -0,0 +1,77 @@
+- Start Date: 2015-06-18
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+Provide clarity and additional elements/properites in API Description namespace.
+
+# Motivation
+
+The purpose of this document is to make it easier for newcomers to understand the format and build tools around it.
+
+# Detailed design
+
+## Combine Namespaces
+
+We should combine the API Description namespace with the resource namespace.
+Right now they are separate and it would be clearer if they were not.
+
+## Elements in Attributes
+
+Refract allows for including elements within the attributes section of an element.
+Even though it’s possible, it may be helpful to avoid this in such an important namespace.
+Reason being, this will be one of the first places people interact with Refract, and to use complex features may be too much.
+
+## HREF Variables
+
+We should call them “hrefVariables” everywhere instead of parameters in places (specifically transitions).
+I think this will be more consistent and easier to understand for newcomers.
+This will make more sense in how we handle inheritance as well, because parameters inheriting from hrefVariables does not seem optimal.
+
+## Defining Classes
+
+Swagger provides a way to define tags that can be applied to resources throughout the document.
+Refract has the “class” attribute that we can use the same way, we only need a way to define a class name and provide the description for it.
+Of course, this won’t carry over to API Blueprint, but that’s OK.
```

Comment:
Swagger has a [tags](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#operation-object). Refract has classes. We can use classes, but have to have a way to define these classes used because they have meta data in Swagger.

The other option is to go the tag route and create tag elements, which is fine for me as well.